dev/RadioButtons/RadioButtons_themeresources.xaml

```diff
@@ -137,8 +148,8 @@
                                 StrokeThickness="{ThemeResource RadioButtonBorderThemeThickness}" />
 
                             <Ellipse x:Name="CheckGlyph"
-                                Width="10"
-                                Height="10"
+                                Width="11"
```

Comment:
I think this should be either 10 (unchanged) or 12 (the border is 1px thinner on each side, so we get 2px extra vertical and horizontal space).

Reply:
Good point

Reply:
I think it would be better if you can implicitly set the CheckGlyph size by using margin or padding. This way the CheckGlyph size will be determined by the outer Ellipse, and we'll always get the correct size even if they customize the border thickness.

Reply:
In order to use implicitly set the glyph size using margin, I would have to move the glyph ellipse to be inside the OuterEllipse for now I just set its size to zero